### ISSUE TITLE
Verify menu structure

### ISSUE THREAD

**Issue by mdoering** [gbif/col-website-ssg#8]

**Comment by dhobern** [comment]
For release one, my suggestion is just:

Lose the Home link - the COL logo does that
Keep just the first four (or even three) items from the Data menu
Perhaps combine some of the new About pages so we have around four in total and include some more basic user guidance
Remove the Resources and Help menus until a later release
News links just to the new blog posts

**Comment by mdoering** [comment]
Should there not be some information about API, supported formats and ColDP, helpdesk, contacts at least?

**Comment by mdoering** [comment]
We also need to list and link the annual archives exposed via docker somewhere - currently resources

**Comment by dhobern** [comment]
Yes - we can add that as we have it, but it is not mandatory for an MVP/beta version. I would tend to put some of this into the About structure for now. Blog posts are another route.
